book/src/404.md

```diff
@@ -0,0 +1,3 @@
+# Document not found (404)
+
+Apologies: This URL is invalid. Try the search instead.
```

Comment:
If we get to the end and its not used we can drop it. Right now i want something that works so I'm following the mdbook structure and setup closely right now

Reply:
I'd prefer to work the other way around: add things when they prove useful/necessary, as otherwise we risk forgetting to remove them later if we don't end up needing them.